### DISCUSSION TITLE
Change focus to a specific program/window over HID?

### DISCUSSION THREAD

**Discussion by ArnieO** [NicoHood/HID#284] [Q&A] [answered]
Is it possible to give focus to a specific computer window via HID?
I have an application where my USB-connected devie sends HID keycodes to a Windows-program. But if the user has changed focus to an other program/window, it obviously does not work as intended.
I would like to change my code so that focus is changed to the right program/window before sending keycodes. Is it possible to do this with HID codes?

**Comment by NicoHood** [comment, marked as answer]
You can only do, what you can do with the keyboard.

I think, that on windows, if you press windows + 1 (2,3,4...) the program inside that taskbar position will be run. So if you pin a program to the taskbar, and program that combination inside the arduino, it could work. But that is something out of scope for this library.

**Reply by ArnieO** [reply]
OK, understood. Could work for a single install but not usable for a solution intended for distribution to various users. 
But thank you for clarifying!
